examples/jupyter_browser_signer.ipynb

```diff
@@ -1,131 +1,41 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": 1,
-   "id": "94744db8",
    "metadata": {},
-   "outputs": [],
+   "cell_type": "markdown",
    "source": [
-    "import boa; from boa.network import NetworkEnv"
-   ]
+    "Before being able to use the plugin, you need to install it. You can do this by running the following command:\n",
+    "```bash\n",
+    "jupyter lab extension enable boa\n",
+    "```\n",
+    "\n",
+    "We provide a multi-user setup with JupyterLab in [try.vyperlang.org](https://try.vyperlang.org/).\n",
+    "The source code for this website is available in [GitHub](https://github.com/vyperlang/try.vyperlang.org).\n",
+    "\n",
+    "It is also possible to run our plugin in [Google Colab](https://colab.research.google.com/).\n",
```

Comment:
maybe just put this in the readme / docs?

Reply:
https://github.com/vyperlang/titanoboa/pull/224/commits/8dfd1b967c0618c6dc08123a312abf9eb127db39